### ISSUE TITLE
Bulk Selection Screen Adjustment Functionality

### ISSUE THREAD

**Issue by ExtCan** [Inferno214221/cmc-mod-manager#3]
First off, I love this tool, it has been SO useful. I do have one problem and that is CSS slots

I love installing new fighters but I have OCD and they need to go with the franchise they're apart of, for example, Toadette (new mod on GB), I want to add her but I can't because I would have to manually go slot by slot moving every fighter one slot forward to add her in the Mario series. 

Please add an option for automated scooting over when I plop a character between two slots. 

I'm sorry if this isn't place for requests, and I'm sorry if my description is hard to follow. 

Thank you!

**Comment by Inferno214221** [comment]
Hi,
Firstly, this is probably the best place to make requests, as they are significantly easier to track than on Discord or GameBanana.
Although I don't have OCD, I also ensured that all characters in my build were sorted on the CSS when I used to play.
I see how this feature could be useful, but it does require some major adjustments to the way the CSS tab currently works. It also raises some questions about what should happen to the other characters which are being moved such as how they should wrap, that are largely dependent on how different players sort their roster.
Rather than implementing this specifically, it might be better to make it possible to select and move multiple characters at once or add options for automatically generating sorted rosters.
Although I think that any of these features would be beneficial to implement, I no longer have very much time that I can dedicate to working on this project and have therefore been focusing my attention primarily on bug fixes. (I am of course happy to accept pull requests if you feel like implementing something yourself.)
Thanks
